### PR TITLE
Remove dependency to S3 files

### PR DESCRIPTION
**Pull request by PysarenkoDS-BWT**
Issue #443 

Currently, running raster analysis locally requires to have a boundary file on S3. We remove this dependency.

Also we can add defaultURL property to each country (for example I add it to Cuba)
if this property exists, then we will receive data from the specified path

For example without it path is `http://localhost:3000/data/myanmar/admin_boundaries.json`
But if we add `defaultUrl: 'prism-cuba.org'` we take next path `prism-cuba.org/data/myanmar/admin_boundaries.json`

### CODE REVIEW

File: src/context/mapStateSlice/index.ts [hunk 1]
```diff
@@ -48,17 +49,29 @@ export const mapStateSlice = createSlice({
...
   reducers: {
     addLayer: ({ layers, ...rest }, { payload }: PayloadAction<LayerType>) => {
-      const layersToAdd = payload?.group?.activateAll
+      const updatedPayload = {
+        ...payload,
+        ...(payload.type === 'boundary' &&
+          countryBaseURL && {
+            path: `${countryBaseURL}/${payload.path}`,
+          }),
+      };
```
Comment (ericboucher) [inline src/context/mapStateSlice/index.ts:58]: can you explain why we're making this change?

Comment (PysarenkoDS-BWT) [reply src/context/mapStateSlice/index.ts:58]: We need this to add baseURL to path field if countryBaseURL available

File: src/context/analysisResultStateSlice.ts [hunk 2]
```diff
@@ -101,10 +99,6 @@ function getAdminBoundariesURL() {
   if (adminBoundariesPath.startsWith('http')) {
     return adminBoundariesPath;
   }
-  // do not send a local path to the API, use a fixed boundary file instead.
-  if (isLocalhost) {
-    return defaultBoundariesPath;
```
Comment (ericboucher) [inline src/context/analysisResultStateSlice.ts:106]: I think this is the only part that should change, actually

File: src/context/analysisResultStateSlice.ts [hunk 3]
```diff
@@ -101,10 +99,6 @@ function getAdminBoundariesURL() {
   if (adminBoundariesPath.startsWith('http')) {
     return adminBoundariesPath;
   }
-  // do not send a local path to the API, use a fixed boundary file instead.
-  if (isLocalhost) {
```
Comment (ericboucher) [inline src/context/analysisResultStateSlice.ts:105]: this will break when the API is being run locally, no?

File: src/config/cuba/index.ts [hunk 4]
```diff
@@ -9,5 +9,6 @@ export default {
   rawLayers,
   rawTables,
   translation,
+  defaultUrl: 'prism-cuba.org',
```
Comment (ericboucher) [inline src/config/cuba/index.ts:12]: This URL does not exist, so why put it in here?

Comment (PysarenkoDS-BWT) [reply src/config/cuba/index.ts:12]: for tests, sorry, forgot to delete it(

File: src/context/analysisResultStateSlice.ts (hunk 3 above)
Comment (PysarenkoDS-BWT) [reply src/context/analysisResultStateSlice.ts:105]: No, all works fine

**Pull request closed without merging**